docs/upgrade/index.md

```diff
@@ -2,13 +2,13 @@
 html_meta:
   "description": "Plone 6 upgrade Guide"
   "property=og:description": "Plone 6 upgrade guide"
-  "property=og:title": "Plone 6 upgrade guide"
+  "property=og:title": "Upgrade guide"
```

Comment:
I would mention here in meta data 'Plone6' cause this is what sharing on Twitter, etc takes. I'm not sure about Google, would be cool if it takes the og:title. Then the title on docs.plone.org can be without mentioning 'Plone 6' again and again, but Links would do mention 'Plone 6'.

Reply:
Done in 0dd64365b205efef1fff3f4df101dd6064fd0c29.